src/components/blog/post/body/_paragraph.scss

```
@@ -1,6 +1,7 @@
 @import "./variables";
 
-p {
+p,
+:global(.custom-paragraph) {
```

Instead of `custom-paragraph` how about `BlogPostParagraph` as a SuitCSS component? I get the idea, but in terms of styles, this is basically what you want, to be able to define that some element is to be seen as a regular paragraph. The fact we use the tags at all here are just a consequence of the Markdown rendering.

```suggestion
:global(.BlogPostParagraph) {
```